pkg/process/checks/container.go

```diff
@@ -156,6 +158,10 @@ func fmtContainers(ctrList []*containers.Container, lastRates map[string]util.Co
 			Addresses:   convertAddressList(ctr),
 			Started:     ctr.StartedAt,
 			Tags:        tags,
+			Conns: &model.ConnsStat{
+				OpenedTcp: connsCounts.TCP,
```

Comment:
Feel free not to do this, but I somehow like them to be direct fields on container struct, so I don't have to do `container.Conns.OpenedTcp`.

Reply:
Another reason is that protobuf would make the nested field a reference not an object, so it's not as GC friendly.

Reply:
This was to avoid repetition on all the different objects (`process_rt`, `process`, `container_rt`, `container`)